packages/ui/src/common/hooks/useSignAndSendTransaction.ts

```diff
@@ -14,6 +14,11 @@ interface Params {
   service: ActorRef<any>
 }
 
+// Transactions which emit events handled by QueryNode use useSignAndSendQueryNodeTransaction hook
+// that waits for QueryNode confirmation on PROCESSING stage.
+// Other transactions use simplified hook useSignAndSendTransaction which automatically switch
+// from state FINALIZING to PROCESSING.
+
```

Comment:
I chose which transactions emit events handled by QueryNode (and therefore need to use useSignAndSendQueryNodeTransaction hook) based on:
https://github.com/Joystream/joystream/blob/olympia/query-node/manifest.yml